plugins/rikki/heroeslounge/models/Match.php

```diff
@@ -82,6 +83,32 @@ class Match extends Model
         ]
     ];
 
+    public function listWinnerOptions($fieldname, $value, $formData)
+    {
+        return [$formData["teams"][0]['id'] => $formData["teams"][0]['title'], $formData["teams"][1]['id'] => $formData["teams"][1]['title']];
+    }
+
+    public function listDivisionOptions($fieldname, $value, $formData)
+    {
+        $divisions = Division::all();
+        $list = [];
+
+        foreach ($divisions as $division) {
+            if (!is_null($division->season_id)) {
```

Comment:
Should this logic live on the division model so that it can be reused elsewhere?  This kind of naming seems like it is not unique to this dropdown, potentially.

Reply:
I'm not sure if it's worth moving this to the division model. However I did find that there's already a method `getLongTitleAttribute()` on the division model. It would probably be worth updating that logic and using it to create the long division title for each division.

Reply:
oh yeah you should definitely modify that method because it is almost exactly the same as this logic already, and it does not appear to be used anywhere...  (So changing it is totally safe and would be perfect here)